scripts/ci/setup-travis.sh

```diff
@@ -9,7 +9,7 @@ else
 fi
 
 if [[ "${DO_COVERAGE}" == "ON" ]]; then
-    export CMAKE_ARGS="$CMAKE_ARGS -DCHFL_CODE_COVERAGE=ON"
+    export CMAKE_ARGS="$CMAKE_ARGS -DCMAKE_C_FLAGS=\"--coverage\" -DCMAKE_CXX_FLAGS=\"--coverage\""
```

Comment:
Do we still need to check for GCC?

Reply:
No, because `$DO_COVERAGE` is only set when using GCC

Reply:
That's CI only I assume.

I guess if the user tries to run coverage on a non-GCC platform their fate is their own now!

Reply:
Yep, that's how I see it =)